areas/admin.py

```diff
@@ -11,16 +10,29 @@ class ContractZoneAdmin(OSMGeoAdmin):
     default_lat = 8438120
     default_zoom = 10
     modifiable = False
-
-    fieldsets = (
-        ("", {"fields": ("contact_person", "phone", "email", "contractor_user")}),
-        (
-            _("Imported fields"),
-            {"fields": ("name", "active", "origin_id", "contractor", "boundary")},
-        ),
+    list_display = ("name", "active")
+    ordering = ("name",)
+    fields = (
+        "name",
+        "boundary",
+        "active",
+        "origin_id",
+        "contractor",
+        "contact_person",
+        "phone",
+        "email",
+        "contractor_user",
     )
 
     def has_add_permission(self, request, obj=None):
         return False
 
-    readonly_fields = ("active", "name", "origin_id", "contractor")
+    readonly_fields = (
```

Comment:
kinda weird that this attribute is not defined with the same attributes, but after the `has_add_permission` method. it was already like that, maybe nobody noticed last time 😅

Reply:
👍